pydra/engine/submitter.py

```diff
@@ -16,24 +17,31 @@
 class Submitter:
     """Send a task to the execution backend."""
 
-    def __init__(self, plugin="cf", **kwargs):
+    def __init__(self, plugin: ty.Union[str, Worker] = "cf", **kwargs):
         """
         Initialize task submission.
 
         Parameters
         ----------
-        plugin : :obj:`str`
-            The identifier of the execution backend.
+        plugin : :obj:`str` or :obj:`pydra.engine.core.Worker`
+            Either the identifier of the execution backend or the backend itself.
             Default is ``cf`` (Concurrent Futures).
 
         """
         self.loop = get_open_loop()
         self._own_loop = not self.loop.is_running()
-        self.plugin = plugin
-        try:
-            self.worker = WORKERS[self.plugin](**kwargs)
-        except KeyError:
-            raise NotImplementedError(f"No worker for {self.plugin}")
+        if isinstance(plugin, Worker):
```

Comment:
Do we want to allow people to pass uninitialized classes, as well as pre-initialized? Here's an approach that looks up a class for `str`, initializes a class for `type`:

```python
if isinstance(plugin, str):
    plugin = WORKERS.get(plugin)
    if plugin is None:
        raise NotImplementedError(f"No worker for {plugin}")
if isinstance(plugin, type):
    plugin = plugin(**kwargs)

self.worker = plugin
self.plugin = plugin.plugin_name
```

Given that we're allowing initialization options, it seems weird not to permit classes, to me. But I can also see a case for keeping the options fewer. Up to you.

Alternately, we could get rid of `kwargs`. People can use defaults or pass a fully initialized `Worker`.

Reply:
Have changed it to a class instead of an instance